person.rb

```diff
@@ -1,18 +1,25 @@
-class Person
+require_relative 'nameable'
```

Comment:
* I get this error when i try to test your decorated class , Kindly fix this error 👇 
![image](https://github.com/stephenkati/oop-school-lib/assets/52497246/0b4b017e-e795-4b31-9c61-023e61fe7798)